compiler/fir/tree/src/org/jetbrains/kotlin/fir/declarations/FirDeclarationUtil.kt

```diff
@@ -107,4 +107,7 @@ fun FirRegularClass.addDeclaration(declaration: FirDeclaration) {
         is FirSealedClassImpl -> declarations += declaration
         else -> throw IllegalStateException()
     }
-}
\ No newline at end of file
+}
+
+object isFromVarargKey: FirDeclarationDataKey()
```

Comment:
Please, make this object private if possible and use capitalized names for objects (https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules)

Reply:
Done. Thanks!